starlite_jwt/middleware.py

```diff
@@ -58,12 +60,16 @@ async def authenticate_request(self, connection: "HTTPConnection") -> Authentica
         Raises:
             [NotAuthorizedException][starlite.exceptions.NotAuthorizedException]: If token is invalid or user is not found.
         """
-        auth_header = connection.headers.get(self.auth_header)
-        if not auth_header:
+        encoded_token = connection.headers.get(self.auth_header)
+
+        if not encoded_token:
             raise NotAuthorizedException("No JWT token found in request header")
+        return await self._authenticate_request(connection=connection, encoded_token=encoded_token)
+
+    async def _authenticate_request(self, connection: "HTTPConnection", encoded_token: "Any") -> AuthenticationResult:
```

Comment:
so instead of a private method, you can have a method called `authenticate_token`, because this method does not require the connection passed to it at all as far as i can see.